src/deconvolution.jl

```diff
@@ -152,7 +152,7 @@ function deconvolution(measured::AbstractArray{T, N}, psf;
                           optim_options=optim_options,
                           mapping=mapping,
                           loss=loss,
-                          debug_f=debug_f)
+                          debug_f=debug_f, use_optim=use_optim)
```

Comment:
I would prefer something like: "optimizer_package" since `use_optim` does not imply other packages

Reply:
OK